source/develop/release-management/features/virt/virt-v2v-integration.html.md

```diff
@@ -139,6 +139,33 @@ Import from external environments using virt-v2v in terms of the general flow:
 
 10. Unlock VM & disks
 
+##### Import OVA from VMware
+
+Currently oVirt support importing OVA only from VMware hypervisor.
+The OVA formats that oVirt supported are tar, zip and folder.
```

Comment:
is zip supported? the ova specifically doesn't include zip (only the disks may be zipped)